packages/atomic/src/components/facets/atomic-segmented-facet-scrollable/atomic-segmented-facet-scrollable.tsx

```diff
@@ -26,21 +26,30 @@ export class AtomicSegmentedFacetScrollable {
   }
 
   private renderArrow(arrowDirection: string) {
-    return (
+    return [
       <Button
         style="square-neutral"
-        class="flex shrink-0 basis-8 justify-center items-center rounded"
+        class={`flex shrink-0 basis-8 justify-center items-center rounded absolute z-10 w-10 top-0 bottom-0 ${
```

Comment:
Same as mentioned in the other PR, I would make sure that every element that has styling also has a part.